packages/mobile/src/screens/sign-up-screen/screens/SelectArtistsScreen.tsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react'
+
+import { getGenres } from 'audius-client/src/common/store/pages/signon/selectors'
+import { Pressable, View } from 'react-native'
+import { useSelector } from 'react-redux'
+
+import { Text } from 'app/components/core'
+
+const messages = {
+  header: 'Follow At Least 3 Artists',
+  description:
+    'Curate your feed with tracks uploaded or reposted by anyone you follow. Click the artist’s photo to preview their music.',
+  continue: 'Continue'
+}
+
+export const SelectArtistsScreen = () => {
```

Comment:
is genre part of select artists, or is this component misnamed?

Reply:
It's confusing but this is the page that receives the selected genres from the select genre page and allows user to view top artists for those genres. This select is to select the genre to display the artist, make sense?